src/components/chat-interface.tsx

```
@@ -11,10 +11,23 @@ import {
 import { Badge } from "@/components/ui/badge";
-import { Brain, MessageCircle, Sparkles, Zap } from "lucide-react";
+import {
+  Heart,
+  MessageCircle,
+  Sparkles,
+  Zap,
+  GitBranch,
+  Network,
+  Users,
+  Share2,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)